test/degree.jl

```diff
@@ -0,0 +1,29 @@
+using Symbolics
+using Test
+
+@variables x, y, z
```

Comment:
We discourage people from writing `@variables x, y, z` so let's not use it in our tests either. The problem is that `@variables x=1, y, z` will be parsed as `@variables x=(1, y, z)` which is obviously not what people would expect.

Reply:
Thanks for letting me know. I just saw it in other tests such as this one https://github.com/JuliaSymbolics/Symbolics.jl/blob/a9eb5b57bb87c1d5a4f16e7efdca553c98f878d1/test/diff.jl#L6

Reply:
This file should also be included in `test/runtests.jl` to actually run in the CI.